lightdash/lightdash
# feat: remove filter warning and clear temp filters

Closes: #12459 

### Description:

Removes the warning about clearing filters. Now when you have non-default filters for a dashbaord, they will be cleared. 

To test:
- Change an existing dashboard fitler or add a temp fitler
- Edit the dashboard. 
- You should see no warning
- The filters should be reset to the values in the saved dashboard. 

### Reviewer actions
